backend/chunker/chunker.go

```diff
@@ -1461,6 +1490,7 @@ func (f *Fs) copyOrMove(ctx context.Context, o *Object, remote string, do copyMo
 			break
 		}
 		chunkSuffix := chunkRemote[len(mainRemote):]
+		fs.Debugf(o, "CHUNK SUFFIX: %s", chunkSuffix)
```

Comment:
Debugging leftover. Let's keep it for now but clean em up before merging to master.

Reply:
@mcalman
"Don't kill him, he's mine"
(c) random villain